File: core/src/main/java/com/cc/core/command/impl/OpenWechatAction.java
package com.cc.core.command.impl;

import com.cc.core.MyApplication;
import com.cc.core.actions.Action;
import com.cc.core.actions.ActionResult;
import com.cc.core.wechat.Wechat;

public class OpenWechatAction implements Action {
    @Override
    public ActionResult execute(Object... args) {
        Wechat.startApp(MyApplication.application());
        return ActionResult.successResult();
    }

    @Override
    public String key() {
        return "openWechat";
    }
}

File: core/src/main/java/com/cc/core/command/impl/OpenWechatAction.kt
package com.cc.core.command.impl

import com.cc.core.ApplicationContext
import com.cc.core.actions.Action
import com.cc.core.actions.ActionResult
import com.cc.core.wechat.Wechat

class OpenWechatAction : Action {
    override fun key(): String? {
        return "openWechat"
    }

    override fun execute(vararg args: Any?): ActionResult? {
        Wechat.startApp(ApplicationContext.application())
        return ActionResult.successResult()
    }
}